sys/net/lora/Kconfig

```diff
@@ -0,0 +1,156 @@
+
+endchoice
+
+choice
+    bool "Spreading factor"
+    default LORA_SF_DEFAULT_SF7
+    help
+        Configure Spreading Factor (SF). SF denotes the amount of spreading code
+        applied to the original data signal. A larger SF increases the time on
+        air, which increases energy consumption, reduces the data rate, and
+        improves communication range. Each step up in spreading factor
+        effectively doubles the time on air to transmit the same amount of data.
+        Refer to country specific air time usage regulations before varying the
+        SF. To calculate air time refer https://www.loratools.nl/#/airtime .
+
+config LORA_SF_DEFAULT_SF6
+    bool "SF6"
+
+config LORA_SF_DEFAULT_SF7
+    bool "SF7"
+
+config LORA_SF_DEFAULT_SF8
+    bool "SF8"
+
+config LORA_SF_DEFAULT_SF9
+    bool "SF9"
+
+config LORA_SF_DEFAULT_SF10
+    bool "SF10"
+
+config LORA_SF_DEFAULT_SF11
+    bool "SF11"
+
+config LORA_SF_DEFAULT_SF12
+    bool "SF12"
+
+endchoice
+
+choice
+    bool "Coding rate"
+    default LORA_CR_DEFAULT_CR_4_5
+    help
+        Configure Coding rate (CR). CR denotes the implementation of forward
+        error correction (FEC). This may be done by encoding 4-bit data with
+        redundancies into 5-bit, 6-bit, 7-bit, or 8-bit. Coding Rate (CR) value
+        need to be adjusted according to conditions of the channel used for data
+        transmission. If there are too many interferences in the channel, then
+        it’s recommended to increase the value of CR. However, the rise in CR
+        value will also increase the duration for the transmission. Refer to
+        country specific air time usage regulations before varying the CR. To
+        calculate air time refer https://www.loratools.nl/#/airtime .
+
+config LORA_CR_DEFAULT_CR_4_5
+    bool "CR 4/5"
+
+config LORA_CR_DEFAULT_CR_4_6
+    bool "CR 4/6"
+
+config LORA_CR_DEFAULT_CR_4_7
+    bool "CR 4/7"
+
+config LORA_CR_DEFAULT_CR_4_8
+    bool "CR 4/8"
+
+endchoice
+
+config LORA_IQ_INVERTED_DEFAULT
+    bool "Enable inverted I/Q mode"
+    help
+        Enable Inverted I/Q while sending LoRa frames. Inverted I/Q allows
+        differentiating sender and receiver messages. In the case of LoRaWAN,
+        gateways receive messages with Inverted I/Q off and send messages with
+        Inverted I/Q on. Nodes read messages with Inverted I/Q on and send
+        messages with Inverted I/Q off. Set this to on if the device is to be
+        used as a gateway.
+
+config LORA_FREQUENCY_HOPPING_DEFAULT
+    bool "Enable Frequency hopping spread spectrum (FHSS)"
+    help
+        Enabling Frequency hopping spread spectrum (FHSS) would imply that a
+        portion of each LoRa packet is transmitted on each hopping channel from
+        a look up table of frequencies managed by the host microcontroller.
+
+config LORA_FREQUENCY_HOPPING_PERIOD_DEFAULT
+    int "Frequency hopping period"
+    default 0
+    depends on LORA_FREQUENCY_HOPPING_DEFAULT
```

Comment:
same as below, I think we don't need a dependency here. There are use cases where this value is set but the FHop is disabled by default.

Reply:
Okay.  It looks obvious to me. Fine, I shall correct it.